Source/WebCore/display/css/DisplayBoxDecorationPainter.cpp

```diff
@@ -1252,8 +1252,7 @@ void BoxDecorationPainter::paintFillLayer(PaintingContext& paintingContext, cons
         op == CompositeOperator::SourceOver ? layer.composite() : op,
         layer.blendMode(),
         DecodingMode::Synchronous,
-        ImageOrientation::Orientation::FromImage,
-        InterpolationQuality::Default
```

Comment:
I wonder if it should use this method: https://searchfox.org/wubkat/source/Source/WebCore/rendering/RenderBoxModelObject.cpp#659 like BackgroundPainter.cpp does.

The code in display/ dead code as we speak though, so it doesn't matter too much.